src/App.js

```diff
@@ -16,23 +16,23 @@ function App({ domElement }) {
   const position = positionClass(domElement.getAttribute("data-placement"));
 
   return (
-    <div
-      onClick={() =>
-        (window.location.href = `https://www.heycater.com/de/catering/${city}/${catererSlug}?utm_source=online-shop-widget`)
-      }
-      className={`shop__button__heycater_app ${position}`}
+    <a
+      href={`https://www.heycater.com/de/catering/${city}/${catererSlug}?utm_source=online-shop-widget`}
+      target="top"
```

Comment:
@LoMatzke please check `target="top"` is a correct attr to jump out of iframes

Reply:
@wivern-co-uk the target="_top" seems correct, just my syntax was not

_The [target](https://www.dofactory.com/html/target) attribute specifies where the linked page or form response will be opened.The target="_top" value opens the page in the top frame which always is the full browser tab/window._

Reply:
@wivern-co-uk question to me is if we want to open the MP in the same window or open new window?